src/server.ts

```diff
@@ -25,6 +26,9 @@ export class InversifyExpressServer  {
     ) {
         this._container = container;
         this._router = customRouter || express.Router();
+        this._routingConfig = {
+            rootPath: "/"
```

Comment:
I'm not sure I've got your point here, do you mean provide default routing config or default root path via optional constructor argument?